File: python-examples/captcha-solving-basic-example/helpers/captcha.py
from intuned_runtime.captcha import wait_for_captcha_solve
from playwright.async_api import Page

@wait_for_captcha_solve(timeout_s=30, settle_period=10, wait_for_network_settled=True)
async def go_to_with_captcha_solve(page: Page, url: str):
    await page.goto(url)

File: typescript-examples/captcha-solving-basic-example/helpers/captcha.ts
import { withWaitForCaptchaSolve } from "@intuned/runtime";
import { Page } from "playwright";

export async function goToWithCaptchaSolve(
  page: Page,
  url: string
): Promise<void> {
  await withWaitForCaptchaSolve(async (page) => await page.goto(url), {
    page,
    timeoutInMs: 30_000,
    settleDurationMs: 10_000,
    waitForNetworkSettled: true,
  });
}
